docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,194 @@
+    "- `shots_per_randomization`: Total number of shots per learning circuit\n",
+    "- `layer_pair_depths`: The circuit depths to us in the learning experiments (measured in number of pairs)\n",
+    "- `twirling_strategy`: The twirling strategy used in the layers of two-qubit twirled gates. The allowed values are:\n",
+    "    - `\"active\"`: only instruction qubits are twirled in each individual twirled layer\n",
+    "    - `\"active-circuit\"`: in each twirled layer, the union of all qubits with an instruction are twirled\n",
+    "    - `\"active-accum\"`: in each twirled layer, the union of all qubits with an instruction in the circuit *up to the current layer* are twirled\n",
+    "    - `\"all\"`: in each twirled layer, *all* qubits in the circuit are twirled\n",
+    "\n",
+    "Below is a simple example showing how to use the `NoiseLearnerOptions` in a `NoiseLeaner` experiment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f35d68a-b459-4f1c-ae44-a01bbe700ed0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Build a larger GHZ circuit\n",
+    "qubits = list(range(10))\n",
+    "num_qubits = len(qubits)\n",
+    "qreg = QuantumRegister(num_qubits, \"q\")\n",
+    "circuit = QuantumCircuit(qreg)\n",
+    "circuit.h(0)\n",
+    "circuit.cx(range(0, num_qubits, 2), range(1, num_qubits, 2))\n",
+    "\n",
+    "# Choose a backend to run on\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy()\n",
+    "\n",
+    "# Transpile the circuit for execution\n",
+    "pm = generate_preset_pass_manager(backend=backend, optimization_level=3)\n",
+    "circuit_to_run = pm.run(circuit_to_learn)\n",
+    "\n",
+    "\n",
+    "# Instantiate a noise learner options object\n",
+    "learner_options = NoiseLearnerOptions(\n",
+    "                    max_layers_to_learn = 3,\n",
+    "                    num_randomizations = 32,\n",
+    "                    twirling_strategy = \"all\"\n",
+    "                )\n",
+    "\n",
+    "# Instantiate a NoiseLearner object and execute the noise learning program\n",
+    "learner = NoiseLearner(mode=backend, options=learner_options)\n",
+    "job = learner.run([circuit_to_run])\n",
+    "result = job.result()"
+   ]
+  }
+ ],
+ "metadata": {
+  "description": "Get started with the noise learning helper program to save the noise models created when executing workloads in Qiskit IBM Runtime",
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3"
+  },
+  "title": "Noise learning helper"
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
I think this page needs to also talk about how you can pass the output of a NoiseLearner call to a subsequent Estimator call. This would, for example, allow you to re-use the same noise model for PEA in a variational workload (inside a session) instead of having to re-learn for each iteration. 

Another use case is that it allows people to introspect / modify the noise model, but I'm not entirely sure why people would want to do that. Perhaps @SamFerracin can elaborate more.

Reply:
@jyu00 maybe for the introspection we can wait until we have a few convenience methods on NoiseLearnerResult, like plotting functionalities? 
People normally do a variety of things, but visualization is always the starting point

Reply:
Sure, we can just talk about re-using it in a variational workload for now (which is a major use case).

Reply:
I've added a section at the end on inputting the noise model to the Estimator primitive. Let me know if you think this is sufficient for now until plotting functionalities are added.